Remotion/Validation/UnitTests/Rules/ObjectValidationRuleTest.cs

```diff
@@ -1,21 +1,128 @@
-﻿using NUnit.Framework;
+﻿using System;
+using System.Collections.Generic;
+using Moq;
+using NUnit.Framework;
+using Remotion.Validation.Results;
+using Remotion.Validation.Rules;
+using Remotion.Validation.Validators;
 
 namespace Remotion.Validation.UnitTests.Rules
 {
   [TestFixture]
   public class ObjectValidationRuleTest
   {
     [Test]
-    [Ignore("TODO RM-5906")]
-    public void Validate ()
+    public void Validate_ReturnsValidationFailures ()
     {
+      Func<int, bool> condition = i => true;
+      var context = new ValidationContext(1);
+
+      var failure1 = new ObjectValidationFailure(
+          15,
+          "15 is not a number",
+          "Localized validation message: 15 is not a number.");
+
+      var failure2 = new ObjectValidationFailure(
+          42,
+          "42 is not a number",
+          "Localized validation message: 42 is not a number.");
+
+      var validatorMock1 = new Mock<IObjectValidator>();
+      validatorMock1.Setup(_ => _.Validate(It.IsAny<ObjectValidatorContext>()))
+          .Returns(new[] { failure1 })
+          .Verifiable();
+
+      var validatorMock2 = new Mock<IObjectValidator>();
+      validatorMock2.Setup(_ => _.Validate(It.IsAny<ObjectValidatorContext>()))
+          .Returns(new[] { failure2 })
+          .Verifiable();
+
+      var rule = new ObjectValidationRule<int>(condition, new[] { validatorMock1.Object, validatorMock2.Object });
+
+      var result = rule.Validate(context);
+      Assert.That(result, Is.EquivalentTo(new[] { failure1, failure2 }));
     }
 
     [Test]
-    [Ignore("TODO RM-5906")]
-    public void IsActive ()
+    public void Validate_TryToValidateNull_ReturnsEmpty ()
     {
-      // consider better name than "IsActive"
+      bool condition (object o)
+      {
+        return o != null;
+      }
```

Comment:
```suggestion
Func<object, bool> condition = o => o != null;
```
Since you have it like this in the other tests